browser/ui/webui/playlist_ui.cc

```diff
@@ -30,9 +36,31 @@ bool PlaylistUI::ShouldBlockPlaylistWebUI(
 }
 
 PlaylistUI::PlaylistUI(content::WebUI* web_ui, const std::string& name)
-    : MojoWebUIController(web_ui) {
-  CreateAndAddWebUIDataSource(web_ui, name, kPlaylistGenerated,
-                              kPlaylistGeneratedSize, IDR_PLAYLIST_HTML);
+    : UntrustedWebUIController(web_ui) {
+  // From MojoWebUIController
+  web_ui->SetBindings(content::BINDINGS_POLICY_MOJO_WEB_UI);
+
+  auto* source =
+      CreateAndAddWebUIDataSource(web_ui, name, kPlaylistGenerated,
+                                  kPlaylistGeneratedSize, IDR_PLAYLIST_HTML);
+
+  // Allow to load untrusted resources.
+  source->OverrideContentSecurityPolicy(
+      network::mojom::CSPDirectiveName::StyleSrc,
+      std::string("style-src chrome-untrusted://resources "
+                  "chrome-untrusted://brave-resources 'unsafe-inline';"));
+  source->OverrideContentSecurityPolicy(
+      network::mojom::CSPDirectiveName::ScriptSrc,
+      std::string("script-src chrome-untrusted://brave-resources "
+                  "chrome-untrusted://resources "
+                  "chrome-untrusted://playlist;"));
```

Comment:
minor: can just write `'self'` instead of `chrome-untrusted://playlist`

Reply:
Cool! thaks. done.